src/test/java/rife/engine/FillBeanSite.java

```diff
@@ -183,4 +156,32 @@ public byte[] useInputStream(InputStream stream)
             }
         });
     }
+
```

Comment:
If this isn't reused, why extract?

Reply:
So you don't have such a long (and thus difficult to follow) method.

Reply:
I think it's more difficult if you need to look for it elsewhere

Reply:
That statement breaks any coding practice I've ever encountered. I'll put it back, but you are dead wrong on the difficulty level. Long methods are **always** difficult to follow for some else than the author, splitting into smaller section is not only good practice, it is also making easier for other people to follow the code.

Reply:
I don't agree with that being a 100% rule. There's many standard coding practices that are imho like that.